charts/argo-cd/values.yaml

```diff
@@ -516,6 +516,12 @@ configs:
     bitbucketUUID: ""
     # -- Shared secret for authenticating Gogs webhook events
     gogsSecret: ""
+    ## Azure DevOps
+    azureDevops:
+      # -- Shared secret username for authenticating Azure DevOps webhook events
+      username: ""
+      # -- Shared secret password for authenticating Azure DevOps webhook events
+      password: ""
```

Comment:
📝 
Ref: https://argo-cd.readthedocs.io/en/stable/operator-manual/webhook/#2-configure-argo-cd-with-the-webhook-secret-optional

<img width="487" alt="image" src="https://github.com/argoproj/argo-helm/assets/30188755/fe2c4d38-140c-436c-abd8-ea3b773ade4f">